src/ControlSystem/ControlErrors/CMakeLists.txt

```diff
@@ -1,11 +1,37 @@
 # Distributed under the MIT License.
 # See LICENSE.txt for details.
 
+set(LIBRARY ControlErrors)
+
+add_spectre_library(${LIBRARY})
+
+spectre_target_sources(
+  ${LIBRARY}
+  PRIVATE
+  SizeControlInfo.cpp
+  SizeControlStateAhSpeed.cpp
+  SizeControlStateDeltaR.cpp
+  SizeControlStateInitial.cpp
+  )
+
 spectre_target_headers(
   ${LIBRARY}
   INCLUDE_DIRECTORY ${CMAKE_SOURCE_DIR}/src
   HEADERS
   Expansion.hpp
   Rotation.hpp
+  SizeControlInfo.hpp
+  SizeControlState.hpp
+  SizeControlStateAhSpeed.hpp
+  SizeControlStateDeltaR.hpp
+  SizeControlStateInitial.hpp
```

Comment:
I agree, having a `Size` directory would be good. I also would like to avoid `Control` in all the names since everything here is in the control system. E.g. having directories `Size` and `Errors` to try and keep names somewhat shorter.

Reply:
Once this PR is merged, I can change the `ControlErrors` directory and namepsace to `Errors` to be consistent with everything else.